client/reader.js

```diff
@@ -74,18 +74,26 @@ rl.on('line', (line) => {
     }
   } else if (line.indexOf('FLEX: ') > -1) {
     address = line.match(/FLEX:.*?\[(\d*?)\] /)[1].trim();
-    if (line.match( /( ALN | GPN | NUM)/ )) {
-      message = line.match(/FLEX:.*?\[\d*\] ... (.*?)$/)[1].trim();
-      trimMessage = message;
-    } else {
-      message = false;
-      trimMessage = '';
+      if (line.match( /1600\/2\/F\/A/ )) {
+       if (line.match( /( ALN | GPN | NUM)/ )) {
+        frag = line.match(/FLEX:.*?\[\d*\] ... (.*?)$/)[1].trim();
+        message = false;
+        trimMessage = '';
+       }
+      }
+      if (line.match( /1600\/2\/C\/A/ )) {
+       if (line.match( /( ALN | GPN | NUM)/ )) {
+          message = line.match(/FLEX:.*?\[\d*\] ... (.*?)$/)[1].trim();
+          trimMessage = frag+message;
+          }
+        }
+      if (line.match( /1600\/2\/K\/A/ )) {
+        if (line.match( /( ALN | GPN | NUM)/ )) {
+        message = line.match(/FLEX:.*?\[\d*\] ... (.*?)$/)[1].trim();
+        trimMessage = message;
+        }
+       }
     }
-  } else {
-  	address = '';
-  	message = false;
-  	trimMessage = '';
-  }
```

Comment:
What was this else block removed for? Doesn't look like it's covered elsewhere. IIRC, this bit was needed for when multimon does weird stuff.